pyClient/README.md

```diff
@@ -45,11 +55,211 @@ and fails to fetch it (or fails to fetch the right version).
+$ cp zeth-key.json.pub <destination>
+```
+
+With these files in place, `zeth` commands invoked from inside this directory
+can perform actions on behalf of Alice.  We call this Alice's *client directory*
+below, and assume that all commands are executed in a directory with these
+files.
+
+## Receiving transactions
+
+The following command scans the blockchain for any new transactions addressed to
+the public key `zeth-key.json.pub`:
+
+```console
+# Check all new blocks for notes addressed to `zeth-key.json.pub`, storing them
+# in the ./notes directory.
+(env)$ zeth sync
+```
+
+Any notes found are stored in the `./notes` directory, as files containing the
+secret data required to spend them.  This forms a very primitive Zeth "wallet".
+
+```console
+# List all notes received by this client
+$ zeth notes
+```
+lists information about all known notes belonging to the current user.
+
+## Mix command
+
+The `zeth mix` command is used to interact with a deployed Zeth instance.  The
+command accepts the following information:
+
+**Input Notes.** Zeth notes owned by the current client, which should be visible
+via `zeth notes`.  Either the integer "address" or the truncated commitment
+value (8 hex chars) can be used to specify which notes to use as inputs.
+
+**Output Notes.** Given as pairs of Zeth public key and value, separated by a
+comma `,`.  The form of the public key is exactly as in the `zeth-key.json.pub`
+file.  That is, two 32 byte hex values separated by a colon `:`.
+
+**Public Input.** Ether or ERC20 token value send to the mixer.
+
+**Public Output.** Ether or ERC20 tokens value to be extracted from the mixer.
+
+Some examples are given below
+
+### Depositing funds
+
+A simple deposit consists of some public input (ether or tokens), and the
+creation of a private note.
+
+```console
+# Deposit 10 ether from `eth-address`, creating a Zeth note owned by Alice
+(env)$ zeth mix --out <public_key>,10 --vin 10
+```
+where `<public-key>` is the contents of `zeth-key.json.pub`.
+
+### Privately send a ZethNote to another user
+
+To privately transfer value within the mixer, no public input / output is
+required.  Unspent notes (inputs) and destination addresses and values are
+specified.
+
+```console
+$ zeth notes
+b1a2feaf: value=200, addr=0
+eafe5f84: value=100, addr=2
+
+$ zeth mix \
+    --in eafe5f84 \                       # "eafe5f84: value=100, addr=2"
+    --in 0 \                              # "b1a2feaf: value=200, addr=0"
+    --out d77f...0e00:cc7c....7f76,120 \  # 120 to this addr
+    --out 3a43...fd3b:9fc8....b838,180    # 180 to this addr
+```
+
+### Withdrawing value from notesa
+
+Specify the note(s) to be withdrawn, and the total value as public output:
+```console
+$ zeth withdraw --in eafe5f84 --vout 100
+```
+
+### Complex Transactions
+
+Public and private inputs and outputs can all be specified for a single
+transaction.  Combining multiple operations can help to obfuscate the meaning of
+the transaction.
```

Comment:
I'd rephrase this section and change the name to something like "A note on the `zeth mix` command", then explain why there is a single command (which seems to be what you do here). 

Maybe something along the lines of:
```
### A note on the `zeth mix` command

As explained above, the `zeth mix` command can be used to deposit funds on the mixer, transfer notes, and withdraw funds from the mixer. Using a single command to achieve all these purpose allows to combine all these operations at will, which greatly improves the privacy level provided by Zeth. In fact, no exact information about the meaning of a transaction is ever leaked to the an observant attacker.

Here are a few examples of complex payments allowed by the Mix function:

$ zeth notes
b1a2feaf: value=200, addr=0
eafe5f84: value=100, addr=2

$ zeth mix \
    --in eafe5f84 \                  # "eafe5f84: value=100, addr=2"
    --vin 5
    --out d77f...0e00:cc7c....7f76,120 \  # 103 to this addr (Bob's address)
    --out 3a43...fd3b:9fc8....b838,180    # 2 to this addr (Someone else's address - can be mine if I need refund)
```

Reply:
Rephrased